emba.sh

```diff
@@ -225,6 +225,7 @@ main()
   export BASE_LINUX_FILES="$CONFIG_DIR""/linux_common_files.txt"
   export PATH_CVE_SEARCH="./external/cve-search/bin/search.py"
   export MSF_PATH="/usr/share/metasploit-framework/modules/"
+  export PHP_SCAN_PATH="./external/iniscan/vendor/bin/iniscan"
```

Comment:
use the variable for the external directory. Otherwise we are running into issues in our docker environment